pkg/controller/migplan/migplan_controller.go

```diff
@@ -423,20 +423,80 @@ func (r *ReconcileMigPlan) planSuspended(plan *migapi.MigPlan) error {
 	return nil
 }
 
-func (r *ReconcileMigPlan) ensureMigrationStateLabels(plan *migapi.MigPlan) error {
+func (r *ReconcileMigPlan) ensureMigplanLabelsForMigMigrations(plan *migapi.MigPlan) error {
 
-	//Add or Update labels for migplan having running migrations
-	err := r.AddUpdateMigrationStateToPlan(plan, migctl.Running)
+	// ensure labels for migplan having running migrations
+	err := r.ensureMigPlanRunningLabel(plan)
+	if err != nil {
+		return liberr.Wrap(err)
+	}
 
+	// ensure labels for migplan having failed migrations
+	err = r.ensureMigPlanFailedLabel(plan)
 	if err != nil {
 		return liberr.Wrap(err)
 	}
+	return nil
+}
 
-	//Add or Update labels for migplan having failed migrations
-	err = r.AddUpdateMigrationStateToPlan(plan, migctl.Failed)
+func (r ReconcileMigPlan) ensureMigPlanRunningLabel(plan *migapi.MigPlan) error {
+	migrationStateInstances := 0
```

Comment:
Can we refactor the var `migrationStateInstances` to `runningMigrations` or `runningInstances` or something similar ?